docs/DeveloperGuide.md

```diff
@@ -348,7 +417,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case skips to step 5.
 
-**Use case: UC03 - Add a class**
+**Use case: UC06 - Clear all students**
+
+**MSS**
+
+1. User requests to clear all students.
+2. Tutor's Pet clears all students.
```

Comment:
Should we keep it consistent and use `delete` instead of `clear`?

Reply:
I think `clear` is used to clear all students though... `delete` is meant to delete a specific student...

Reply:
Will keep clear